pyface/action/menu_bar_manager.py

```diff
@@ -8,7 +8,11 @@
 #
 # Thanks for using Enthought open source!
 
-""" A menu bar manager realizes itself in errr, a menu bar control. """
+""" A toolkit-specific menu bar manager that realizes itself in a menu bar
+control.
+
+- :attr:`~.MenuBarManager`
+"""
```

Comment:
When building the docs, these "- :attr:`~.MenuBarManager`" lines just show up as 

- MenuBarManager

Without any link.  Not a suggested change now, but maybe as a future improvement, for toolkit specific things we could link to both implementations somehow?  e.g. one link to docs for https://github.com/enthought/pyface/blob/master/pyface/ui/qt4/action/menu_bar_manager.py and another for wx

Reply:
> Without any link.

There is already an issue: https://github.com/enthought/pyface/issues/755
Note that Qt and Wx implementations don't necessarily have exactly the same definition. Cherry-picking the ones I know:
The enum definition for DataViewWidget looks like this in Qt:
https://github.com/enthought/pyface/blob/3ea7832cfaa076eacec2b2ccbab07589fe599893/pyface/ui/qt4/data_view/data_view_widget.py#L100-L104

But it looks like this in Wx:
https://github.com/enthought/pyface/blob/3ea7832cfaa076eacec2b2ccbab07589fe599893/pyface/ui/wx/data_view/data_view_widget.py#L42-L46

We can't merge the two into one without having to pick one over another.
Another hard problem is the dependency on the toolkit while generating the doc. Some mocking (in doc config) may be needed, but that is not necessarily trivial either.